Geometry_Engine/Query/Normal.cs

```diff
@@ -97,17 +107,20 @@ public static List<Vector> Normals(this Mesh3D mesh)
         /**** Public Methods - Curves                   ****/
         /***************************************************/
 
+        [Description("Returns the normal vector to the curve's plane, oriented according to the rigth hand rule. Works only for closed and planar curves.")]
```

Comment:
I would apply it here and to all other methods taking curves.
```suggestion
        [Description("Returns the normal vector to the plane of a given curve, oriented according to the right hand rule. Works only for closed, planar curves.")]
```